engine/core/src/main/java/org/datacleaner/metamodel/datahub/utils/DatahubColumnBuilder.java

```diff
@@ -0,0 +1,130 @@
+/**
+ * DataCleaner (community edition)
+ * Copyright (C) 2014 Neopost - Customer Information Management
+ *
+ * This copyrighted material is made available to anyone wishing to use, modify,
+ * copy, or redistribute it subject to the terms and conditions of the GNU
+ * Lesser General Public License, as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this distribution; if not, write to:
+ * Free Software Foundation, Inc.
+ * 51 Franklin Street, Fifth Floor
+ * Boston, MA  02110-1301  USA
+ */
+package org.datacleaner.metamodel.datahub.utils;
+
+import org.apache.metamodel.schema.Column;
+import org.apache.metamodel.schema.ColumnType;
+import org.apache.metamodel.schema.ImmutableColumn;
+import org.apache.metamodel.schema.Table;
+import org.datacleaner.metamodel.datahub.DatahubTable;
+
+
+public class DatahubColumnBuilder {
```

Comment:
I would suggest to simply use MutableColumn for this. It seems like overkill to create ColumnBuilder for this specific connector. If we need one it could be generic I think, but normally we don't even need it - we just use MutableColumn.